public/index.css

```diff
@@ -322,11 +322,22 @@ code {
 	word-break: break-word;
 }
 
-/*RTL Fix Code dir*/
+/*RTL Fix Code dir for inline code*/
 [dir='rtl'] code {
 	unicode-bidi: plaintext;
 }
 
+/*RTL Fix codeBlock base in Shiki output*/
+[dir="rtl"] span.line {
+    direction: ltr;
+    unicode-bidi: isolate;
+}
+
+/*RTL Fix Arrows*/
+[dir="rtl"] #on-this-page-heading ~ svg, [dir="rtl"] [rel="next"] svg {
+    transform: rotateY(180deg);
+}
```

Comment:
Tested this real quick in DevTools, changing the specificity seems to fix the problem.
```suggestion
[dir="rtl"] .toc-mobile-container svg, [dir="rtl"] [rel="next"] svg {
    transform: rotateY(180deg);
}
```